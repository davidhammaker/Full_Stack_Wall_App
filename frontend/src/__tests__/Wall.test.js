import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Wall from '../components/wall';


configure({ adapter: new Adapter() });


it('displays page correctly', () => {
  const wall = renderer.create(
    <Wall />
  );
  let tree = wall.toJSON();
  expect(tree).toMatchSnapshot();
});
