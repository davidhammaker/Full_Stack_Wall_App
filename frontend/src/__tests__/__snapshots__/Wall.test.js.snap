// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays page correctly 1`] = `
Array [
  <div
    className="btn-group float-right bg-dark mr-2 rounded sm"
  >
    <a
      className="text-light text-decoration-none"
      href="/login"
    >
      <button
        className="btn btn-dark"
      >
        Log In
      </button>
    </a>
    <a
      className="text-light text-decoration-none"
      href="/signup"
    >
      <button
        className="btn btn-dark"
      >
        Sign Up
      </button>
    </a>
  </div>,
  <h1
    className="display-4 p-4 border border-left-0 border-top-0 border-right-0"
  >
    WallÂ App
  </h1>,
  <div
    className="font-italic mb-2 d-flex justify-content-center"
  >
    Please log in to post on the wall.
  </div>,
  <div />,
  <div
    className="font-italic pb-4 mb-2 d-flex justify-content-center"
  >
    Nothing has been posted yet.
  </div>,
]
`;
